.circleci/config.yml

```diff
@@ -1160,6 +1655,236 @@ workflows:
+          requires:
+            - install-grabpl
+      - build-enterprise-frontend:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - codespell:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - lint-go:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - shellcheck-grabpl:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - test-backend:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+            - lint-go
+      - test-frontend:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - mysql-integration-test:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - postgres-integration-test:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+      - package-oss:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+            - build-oss-backend-armv6
+            - build-oss-backend-armv7
+            - build-oss-backend-armv7-musl
+            - build-oss-backend-arm64
+            - build-oss-backend-arm64-musl
+            - build-oss-backend-osx64
+            - build-oss-backend-win64
+            - build-oss-backend-linux-x64
+            - build-oss-backend-linux-x64-musl
+            - build-oss-frontend
+            - test-backend
+            - test-frontend
+            - codespell
+            - shellcheck-grabpl
+            - mysql-integration-test
+            - postgres-integration-test
+      - package-enterprise:
+          filters: *filter-only-master
+          requires:
+            - install-grabpl
+            - build-enterprise-backend-armv6
+            - build-enterprise-backend-armv7
+            - build-enterprise-backend-armv7-musl
+            - build-enterprise-backend-arm64
+            - build-enterprise-backend-arm64-musl
+            - build-enterprise-backend-osx64
+            - build-enterprise-backend-win64
+            - build-enterprise-backend-linux-x64
+            - build-enterprise-backend-linux-x64-musl
+            - build-enterprise-frontend
+            - test-backend
+            - test-frontend
+            - codespell
+            - shellcheck-grabpl
+            - mysql-integration-test
+            - postgres-integration-test
+      - build-oss-windows-installer:
+          filters: *filter-only-master
+          requires:
+            - package-oss
+      - release-next-packages-grabpl:
+          filters: *filter-only-master
+          requires:
+            - test-frontend
+      - publish-oss-master:
+          filters: *filter-only-master
+          requires:
+            - package-oss
+            - build-oss-windows-installer
+      - docker-oss-master:
```

Comment:
I think breaking apart the docker builds to multiple jobs would make a lot of sense too.

Reply:
@xlson You mean to break them up based on architecture? I think this workflow only builds AMD64.

Reply:
Okay. Yes that is what I meant. Building docker images for all the different architectures takes quite a lot of time

Reply:
@xlson OK thanks for the clarification.

Reply:
@xlson Having taken a closer look, I think Docker images may be built for other architectures as well after all. Do you mind if we take care of dividing the build by architecture in a follow-up PR though? Doing that would take some work, so I'd rather have this PR merged first so we can test the new master pipeline.

Reply:
This can definitely wait until later, no need to do everything at once.